### ISSUE TITLE
Add basic configuration options for overlay timer

### ISSUE THREAD

**Issue by jmakhack** [jmakhack/hydrate-reminder#81]
## Feature Reasoning

Not everyone would like to use the hydrate reminder overlay timer implemented as part of #65.\
Some users may want to use the overlay timer but would like it placed in a different location in the client.\
As a result, configuration options should be available for this feature.

## Feature Description

There should be two main configuration options for the new overlay timer feature:
1. Ability to enable/disable
2. Ability to choose location of the overlay

For the overlay location, investigation needs to be done to see what options are available.
At a minimum, the upper left corner and directly above chatbox should be supported.

## Possible Alternatives

If the configuration setting for setting the overlay location is not feasible for any reason, just implementing the enable/disable configuration setting is fine.

## Additional Context

This enhancement is dependent on #65. Work on this should not be started until #65 is completed and closed.
The overlay should be disabled by default and the location should be upper left corner by default.

**Comment by jmakhack** [comment]
As an added bonus, it would be cool to include a configurable option for the color of the timer text in the overlay widget. This is optional for the completion of this enhancement but a nice to have if it is a possible setting we could add.

**Comment by jmakhack** [comment]
Opened PR #136 which adds several configuration options for the overlay timer such as:
1.     Enable/Disable timer overlay
2.     Add ability to select text color on overlay
3.     Add images to select from to display on overlay

I will open a separate issue to look into and implement the configuration option that decides the location of the overlay timer on the UI as described in Part 2 of the Feature Description above.
